Assets/HoloToolkit/SpatialMapping/Materials/SpatialMappingTap.mat

```diff
@@ -0,0 +1,151 @@
+%YAML 1.1
```

Comment:
@pinkwerks I don't see the actual SpatialMappingTap.shader added in this pull request.
Could you please add it? I can help with GitHub if needed.